app/assets/javascripts/media_player_wrapper/avalon_player_new.es6

```diff
@@ -284,32 +284,24 @@ class MEJSPlayer {
     // Build quality
     this.player.buildquality(this.player, null, null, this.mediaElement);
 
-    this.reInitializeCaptions(playlistItemsT);
+    this.reInitializeCaptions();
 
-    this.player.load();
+    // this.player.load();
```

Comment:
Is this load not necessary?

Reply:
I've commented it out by accident. I will fix it.